.prettierignore

```diff
@@ -0,0 +1,4 @@
+# This file specifies files that Prettier should not format
+
+# Skip Lunr JavaScript files
+resource/**/*.js
```

Comment:
The only lunr.js file is `resource/lunr.js` - the others are mine.

```suggestion
resource/lunr.js
```

Should probably add this to `.gitattributes` too:

```
resource/lunr.js linguist-vendored
```